lib/routes/account.js

```diff
@@ -442,15 +452,19 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
+        function sendVerifyLoginEmail() {
+          if (wantsKeys(request)) {
```

Comment:
The more I think about this feature, the more I feel we should consider sending emails on *every* login, whether the keys are requested or not, until we can use some foo to determine whether a login is "safe." The reasoning - an attacker could figure out the user's email/password combo and then sign in as the user to OAuth sites, and the user would be none the wiser. As the sites we are protecting become higher value, there is a greater incentive for us to keep that data safe.

@rfk, @ryanfeeley, @vbudhram, @philbooth - thoughts?

Obviously, outside of the scope of this PR.